backend/tournesol/tests/test_ml_run.py

```diff
@@ -14,10 +14,46 @@ def setUp(self) -> None:
+            status=EmailDomain.STATUS_REJECTED
+        )
+        user1 = UserFactory(email="user1@verified.test")
+        user2 = UserFactory(email="user2@verified.test")
+
+        self.video1 = VideoFactory()
+        self.video2 = VideoFactory()
+
+        ComparisonCriteriaScoreFactory(comparison__user=user1, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=10)
+        ComparisonCriteriaScoreFactory(comparison__user=user2, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=10)
+
+        not_trusted_user1 = UserFactory(email="not_trusted_user1@not_verified.test")
+        not_trusted_user2 = UserFactory(email="not_trusted_user2@not_verified.test")
+        not_trusted_user3 = UserFactory(email="not_trusted_user3@not_verified.test")
+        not_trusted_user4 = UserFactory(email="not_trusted_user4@not_verified.test")
+        not_trusted_user5 = UserFactory(email="not_trusted_user5@not_verified.test")
+        not_trusted_user6 = UserFactory(email="not_trusted_user6@not_verified.test")
+        not_trusted_user7 = UserFactory(email="not_trusted_user7@not_verified.test")
+        not_trusted_user8 = UserFactory(email="not_trusted_user8@not_verified.test")
+        not_trusted_user9 = UserFactory(email="not_trusted_user9@not_verified.test")
+        not_trusted_user10 = UserFactory(email="not_trusted_user10@not_verified.test")
+
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user1, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user2, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user3, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user4, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user5, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user6, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user7, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user8, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user9, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+        ComparisonCriteriaScoreFactory(comparison__user=not_trusted_user10, comparison__entity_1=self.video1, comparison__entity_2=self.video2, score=-10)
+
     def test_ml_train(self):
```

Comment:
**questions / suggestions**

@farnots @amatissart @lfaucon 

Do you think it's relevant here to split this method into two tests instead of one?

Also the current name, `test_ml_train` is very generic and doesn't tell what the test is going to check. I don't know if it's relevant because I'm not familiar enough with this part of the code yet, but I would suggest to change it to make the code more readable and welcoming for new developers. The test case class is already named `TestMlTrain` so we know we are going to test the machine learning algorithm, and choosing a different name may encourage developers to add different tests in the future instead of adding all assertions into a single big test.

What about `test_computed_scores` and `test_computed_scores_untrusted_users`? Or something else.

Reply:
> Do you think it's relevant here to split this method into two tests instead of one?

How do you want to split this method ? The test is to check information before and after the `ml_train` commands. If we split it the tests will be less interesting.

Reply:
> How do you want to split this method ?

Actually I don't know. My question was motivated by the fact we have here one test case with only one test, and I wanted to know if this was intended and relevant or if this test was actually checking two different and independent use cases that could be split in two different tests.

If it's not relevant to split, it's good as is then :+1: